main/zoe/api/contract-support/ratio-math.md

```diff
@@ -80,23 +88,24 @@ Throws messages for errors:
 assertIsRatio(aRatio);
 ```
 
-## `multiplyBy(amount, ratio)`
+## `floorMultiplyBy(amount, ratio)`
+## `ceilMultiplyBy(amount, ratio)`
 - `amount`: `{ Amount }`
 - `ratio`: `{ Ratio }`
 - Returns: `{ Amount }`
 
 Returns an immutable `Amount`.  Its brand is the `ratio`'s *numerator*'s brand.
-Note the denominator brand has to be the same as the amount brand
+Note the denominator brand has to be the same as the amount brand.
 
 The resulting value is determined by:
 1. Multiplying the `amount` value by the `ratio`'s numerator's value.
 2. Dividing the result from step 1 by the `ratio`'s denominator's value.
-3. Applying a floor to the result from step 2 to round it down to
-   the nearest integer. If the step 2 result is already an integer, its value does
-   not change.
-   
-The above describes what is done to determine the result, not precisely how it's
-done in the source code. 
+3. If that results in an integer, that value is returned, otherwise, the value
+  is rounded down (for floorMultiplyBy) or up (for ceilMultiplyBy) to the next
+  integer.
+
+  The above describes what is done to determine the result, not precisely how
+  it's done in the source code.
```

Comment:
I realize this comment isn't new, but it seems strange to me that we can't describe how it's done in the source code. A user may want to know

Reply:
The step-by-step does describe the source code. I think the old version was vaguer, and that's the reason the caveat was there originally. I've dropped the caveat.